ci-operator/config/openshift/containernetworking-plugins/openshift-containernetworking-plugins-master.yaml

```diff
@@ -11,12 +11,9 @@ build_root:
     tag: golang-1.10
 canonical_go_repository: github.com/openshift/containernetworking-plugins
 images:
-- dockerfile_path: images/supported/Dockerfile
+- dockerfile_path: Dockerfile
   from: base
-  to: container-networking-plugins-supported
-- dockerfile_path: images/unsupported/Dockerfile
-  from: base
-  to: container-networking-plugins-unsupported
+  to: container-networking-plugins
```

Comment:
You also need to change release-4.3 and release-4.2 versions

Reply:
Thanks, got a new commit pushed up with changes to the `-release-4.2.yaml` and `-release-4.3.yaml` files